runtime/src/bank/bank_hash_details.rs

```diff
@@ -23,9 +23,51 @@ use {
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub(crate) struct BankHashDetails {
-    /// client version
+    /// The client version
     pub version: String,
+    /// The encoding format for account data buffers
     pub account_data_encoding: String,
+    #[serde(rename = "banks")]
```

Comment:
Not sure I love `banks` here, maybe just stick with `bank_hash_details`. Don't necessarily like `slots` or `blocks` any more

Reply:
ya i agree. don't love `banks` since that's not really what it is. I think you're right in maybe just sticking with `bank_hash_details`